README.md

```diff
@@ -91,14 +115,99 @@ For **Azure CLI**, create a `public.json` (and optionally `protected.json`) and
+ * Find StorageAccountKey:	 
+  * `$StorageAccount = Get-AzureRmStorageAccount -StorageAccountName $StorageAccountName  -ResourceGroupName $rgName` # Get the Storage Account 
+  * `$StorageAccountKey = (Get-AzureRmStorageAccountKey -ResourceGroupName $rgName -StorageAccountName $StorageAccountName).Value[0]` # Get the primary Key
+  * `$StorageAccountContext = New-AzureStorageContext -StorageAccountKey $StorageAccountKey -StorageAccountName $StorageAccountName` # Get the Context
+  * `$UriEndpoint = $StorageAccountContext.BlobEndPoint`  # EndPoint URL
+  * Grab the endpoint URL
+  - `$filename = "hello.sh"`
+  - `$ConfigScriptUri = $StorageAccountContext.BlobEndPoint + "<container>/"+$fileName`
+  - `$ConfigScriptFileName =  $fileName`
+
+```
+$Location = "centralus"
+$rgName = "<ResourceGroupName>"
+$VmName = "<vm-name>"
+$ExtensionName = "CustomScriptForLinux"
+$Publisher = 'Microsoft.OSTCExtensions'
+$Version = "1.5"  # Latest version https://github.com/Azure/custom-script-extension-linux
+
+# Public Configuration ... from above
+$ConfigScriptURI="https://gist.github.com/ahmetalpbalkan/b5d4a856fe15464015ae87d5587a4439/raw/466f5c30507c990a4d5a2f5c79f901fa89a80841/hello.sh"
+$ConfigScriptFileName = "hello.sh"
+$Command2Exec = "sh $ConfigScriptFileName"
+$PublicConf = '{
+   "fileUris": ["'+$ConfigScriptUri+'"],
+   "commandToExecute": "'+$Command2Exec+'"
+}'
+$StorageAccountName = "<storage-account-name>"
+$StorageAccountKey = "<storage-account-key>"
+$PrivateConf = '{
+     "storageAccountName": "'+$StorageAccountName+'",
+     "storageAccountKey": "'+$StorageAccountKey+'"
+}' 
+
+# Backtick is the character being used below to continue on next line (uaually left of the 1 on QWERTY keyboard)
+# We have all the variables set, let's execute....
+
+Set-AzureRmVMExtension -ResourceGroupName $rgName -VMName $VmName -Location $Location `
+  -Name $ExtensionName -Publisher $Publisher `
+  -ExtensionType $ExtensionName -TypeHandlerVersion $Version `
+  -Settingstring $PublicConf -ProtectedSettingString $PrivateConf
+
+  # Now tell the users where the files are located...
+Write-Host "Your Execution Script files are downloaded to: /var/lib/waagent/$Publisher.$ExtensionName-$version.?.?/#/" -ForegroundColor Yellow
+Write-Host "    command output is saved to stdout and stderr files in this directory" 
+Write-Host "You can find the logs for the extension: "
+Write-Host "     /var/log/azure/$Publisher.$ExtensionName/$version/CommandExecution.log " -ForegroundColor Green
+Write-Host "     /var/log/azure/$Publisher.$ExtensionName/$version/extension.log" -ForegroundColor Green
+  
+```
+
 # 3. Troubleshooting
 
-Your files are downloaded to a path like: `/var/lib/waagent/custom-script/download/0/` and
+Your files are downloaded to a path like: 
+   `/var/lib/waagent/<Publisher>.<ExtensionName>-<version>/#/ScriptName.ext` 
+    Example: 
+	  `/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.5.2.1/download/0/hello.sh` 
```

Comment:
This path doesn't sound correct. You're using the old extension at https://github.com/Azure/azure-linux-extensions repo :)